### PR TITLE
LIR Parcels

### PR DESCRIPTION
**Pull request by miriamseely**
Current metadata draft for the LIR Parcels (we have a separate metadata document for the [Basic Parcels](https://github.com/agrc/metadata-asset-tracking-tool/blob/main/metadata/cadastre/utah-parcels.md)). This pull request also includes a document for the field definitions for this layer.

### CODE REVIEW

File: metadata/Cadastre/lir_parcels_schema.md
Comment (rkelson) [inline metadata/Cadastre/lir_parcels_schema.md:1]: the third page of this file contains the attribute descriptions we should use:
https://drive.google.com/file/d/0BxZfA5vHEKqzZC1RSWI4SjdtQnM/view?usp=sharing&resourcekey=0-3ztJ5Di0zTPnIgplfQRKKA

ExpandedParcelDataSharingImplementationGuidelinesDRAFT.pdf

File: metadata/Cadastre/lir_parcels.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+<!--- This is the ID for Parcels, which at the moment is the same for both Basic Parcels and LIR Parcels. May need to be changed later when we have a new ID for LIR Parcels. -->
+
+## Brief Summary
+
+Land Information Records (LIR) Parcel boundaries for all 29 counties in Utah that include a wide variety of attributes beyond the basic information found in Utah Basic Parcels.
+
+## Summary
+
+This dataset contains information such as the county FIPS code, parcel ID, parcel address, generalized land ownership type, currency dates, total market value, tax district, phone number for the county recorder, and the county parcel website if available. Landowners of privately owned parcels are not specified.
```
Comment (rkelson) [inline metadata/Cadastre/lir_parcels.md:17]:
```suggestion
This dataset contains information such as parcel ID, serial number, parcel address, tax exempt type, tax district, land market value, total market value, parcel acres, property class, building square footage, number of floors, and year built.
```

Comment (steveoh) [reply metadata/Cadastre/lir_parcels.md:17]: If this is a list of every attribute we should remove the `such as` prefix which to implies a subset.

**Review by rkelson — COMMENTED** [review]
Looking at this a bit closer, there is a ton of information available in the Open Data Details of the LIR parcels. I suggest referring to [this layer](https://opendata.gis.utah.gov/datasets/utah::utah-utah-county-parcels-lir/about) for LIR metadata content.

File: metadata/Cadastre/lir_parcels.md (hunk 1 above)
Comment (rkelson) [reply metadata/Cadastre/lir_parcels.md:17]: great observation and you are correct, it is an incomplete list and highlights what are generally populated and I chose to list out.

File: metadata/Cadastre/lir_parcels_schema.md
Comment (miriamseely) [reply metadata/Cadastre/lir_parcels_schema.md:1]: Fantastic, I'll add these in!

**Pull request merged**